site/app/controllers/student/SubmissionController.php

```diff
@@ -697,6 +697,7 @@ public function ajaxUploadSplitItem($gradeable_id, $merge = null, $clobber = nul
             "course" => $this->core->getConfig()->getCourse(),
             "gradeable" => $gradeable->getId(),
             "required_capabilities" => $gradeable->getAutogradingConfig()->getRequiredCapabilities(),
+            "generate_output" => false,
```

Comment:
Let's make this an optional field.  I would rather not change the queue file for non generated output gradeables.  Only put this field in the queue files for building a gradeable with generated output.

If installed on a server that had a backlog of regular autograding, this would be a breaking change.

Also if this field is required, it will mean we need to change the batch regrade python script.  (Maybe you did that.. 
 haven't finished reviewing).